<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadLayerDockWidgetBase</class>
 <widget class="QDockWidget" name="LoadLayerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loadlayer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="loadRotterdamButton">
      <property name="text">
       <string>Load sample</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="showCoordinates"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="selectPoint">
      <property name="text">
       <string>Choose point</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
